src/challenges/challenge_data.cpp

```diff
@@ -79,7 +80,46 @@ ChallengeData::ChallengeData(const std::string& filename)
         return;
     }
 
+    const XMLNode* unlock_list_node = root->getNode("unlock_list");
+    if (unlock_list_node == NULL)
+    {
+        throw std::runtime_error("Challenge file " + filename +
+                                 " has no <unlock_list> node!");
+    }
+
+    std::string list;
+    unlock_list_node->get("list", &list);
+
+    if(list=="true")
+        m_is_unlock_list = true;
```

Comment:
could be written in one line : m_is_unlock_list = (list=="true");

I still don't quite understand what "unlock list" means